### PR TITLE
[#545] add missing constructor of WalletUtils by using fullWalletFile default

### LINKED ISSUE #545: Missing constructor of WalletUtils with 2-param password and location
As the usage from https://docs.web3j.io/transactions.html#creating-and-working-with-wallet-files

> String fileName = WalletUtils.generateNewWalletFile(
        "your password",
        new File("/path/to/destination"));

### PR DESCRIPTION
**Pull request by hackty**
fix #545
add missing constructor of WalletUtils by using lightWalletFile default
add and optimize the testcase for it

### CODE REVIEW

**Comment by conor10** [comment]
I don't understand why we need this method? Why do we need a default?

File: crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java [hunk 1]
```diff
@@ -51,16 +51,23 @@ public void testGenerateBip39Wallets() throws Exception {
     @Test
     public void testGenerateFullNewWalletFile() throws Exception {
         String fileName = WalletUtils.generateFullNewWalletFile(PASSWORD, tempDir);
-        testGenerateNewWalletFile(fileName);
+        testGeneratedNewWalletFile(fileName);
     }
 
+    @Test
+    public void testGenerateNewWalletFile() throws Exception {
+        String fileName = WalletUtils.generateLightNewWalletFile(PASSWORD, tempDir);
```
Comment (conor10) [inline crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java]: We should default to the full wallet file - see https://github.com/ethereum/go-ethereum/blob/8f8774cf6dd5bbcba9fa3773fa34d13d6523a147/node/config.go#L376

Comment (hackty) [reply crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java]: Well, it has been changed default to the full wallet file.

Comment (hackty) [reply crypto/src/test/java/org/web3j/crypto/WalletUtilsTest.java]: Well, it has been changed as the full wallet file by default.

**Pull request merged**